internal/service/account/sweep.go

```diff
@@ -1,5 +1,4 @@
 //go:build sweep
-// +build sweep
```

Comment:
why would we want to remove this directive?

Reply:
That is for older [versions](https://stackoverflow.com/a/68361186).